prow/e2e_install.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+ROOT=$(dirname "$WD")
+export GO111MODULE=on
+
+cd "${ROOT}"
+echo "installing istio with operator CLI"
+go run ./cmd/mesh.go manifest apply
+
+ISTIO_DIR=$(mktemp -d)/src/istio.io/istio
```

Comment:
I would recommend just looking in `$GOPATH/src/istio.io/istio`. Prow can check out additional repos for you so you don't need to `git clone` by passing `extra_refs`